assignments/objects.js

```diff
@@ -19,27 +19,83 @@ let example = {
 
 // Write your intern objects here:
 
+let intern1 = { 
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F" 	
+};
+
+
+let intern2 = {
+ "id": 2,
+ "name": "Kennan",
+ "email": "kdiben1@tinypic.com",
+ "gender": "M"
+};
+
+let intern3 = {
+ "id": 3,
+ "name": "Keven",
+ "email": "kmummery2@wikimedia.org",
+ "gender": "M"
+};
+
+let intern4 = {
+ "id": 4,
+ "name": "Gannie",
+ "email": "gmartinson3@illinois.edu",
+ "gender": "M"
+};
+
+let intern5 = {
+ "id": 5,
+ "name": "Antonietta",
+ "email": "adaine5@samsung.com",
+ "gender": "F"
+};
+
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+intern1["name"];
 
 // Kennan's ID
+intern2["id"];
 
 // Keven's email
+intern3["email"];
 
 // Gannie's name
+intern4["name"];
 
 // Antonietta's Gender
+intern5["gender"];
+
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+intern2.speak = function(){
+  return "Hello, my name is Kennan";
+};
+
+ console.log(intern2.speak());
+
+
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
 //console.log(antonietta.multiplyNums(3,4));
 
+intern5.multiplyNums = function(a,b){
+  return (a*b);
```

Comment:
This is technically accurate, however, its best practices to leave spaces between elements.  i.e. return (a * b).